docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -476,7 +476,109 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After calling `set_time_index`, the `time_index` semantic tag has been added to the semantic tags for `times` column."
+    "After calling `set_time_index`, the `time_index` semantic tag has been added to the semantic tags for `times` column.\n",
+    "\n",
+    "## Validating Woodwork's Typing Information\n",
+    "\n",
+    "The logical types, physical types, and semantic tags described above make up a DataFrame's typing information, which will be referred to as its \"schema\". For Woodowork to be useful, the schema must be valid with respect to its DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Woodwork schema shown above can be seen reflected in the DataFrame below. Every column present in the schema is present in the DataFrame, the dtypes all match the physical types defined for each column, and the index column is both unique and represented on the DataFrame's underlying index."
```

Comment:
Wondering if we should say "...the dtypes all match the physical types defined by each column's LogicalType..." here instead to make it a little more clear the physical type definition is not specified separate from the LogicalType?

Reply:
Also for the index, maybe we could say "...and the Woodwork index column is both unique and matches the DataFrame's index."

Reply:
what do you think of saying `matches the DataFrame's underlying index` to follow the dichotomy of "Woodwork index" vs "DataFrame's underlying index", hopefully avoiding confusion

Reply:
Yeah, that sounds good.